nullaway/src/test/java/com/uber/nullaway/VarargsTests.java

```diff
@@ -71,6 +71,39 @@ public void testNullableVarargs() {
         .doTest();
   }
 
+  /** Test for a @Nullable declaration annotation on a varargs parameter defined in bytecode */
+  @Test
+  public void nullableDeclarationVarArgsFromBytecode() {
+    defaultCompilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import com.uber.lib.Varargs;",
+            "public class Test {",
+            "  public void testDeclaration() {",
+            "    String x = null;",
+            "    Varargs s = new Varargs(x);",
```

Comment:
Can we add the case passing `new Varargs(x, y)` where `x` is non-null but `y` is `@Nullable`?

Reply:
0abca22a59be3e96f88cf7bb785f040a44956641